src/css/variables.css

```diff
@@ -53,7 +54,7 @@
   --branding-font-size:             1.50em; /* the title, e.g., "itwêwina" */
   --branding-subtitle-font-size:    0.75em; /* the subtitle, e.g., "Plains Cree Dictionary" */
   --definition-title-size:          2.00em;
-  --tooltip-icon-size:              2rem; /* size of the question mark or the exclamation mark. Kept the same across the app */
+  --tooltip-icon-size:              0.75em; /* size of the ℹ️  button or 🔊 button. Kept the same across the app */
```

Comment:
```suggestion
  --tooltip-icon-size:              0.75em; /* size of the ℹ️  button or 🔊 button */
```

I used `em` at first too, then it turned out the icon beside the entry title and the preverb breakdown has different sizes. It looked kinda bad cuz they are not uniform. So I changed it to `rem` and added _Kept the same across the app_

Reply:
but it's ok if em produces the same icon size, or it's simply different aesthetics

Reply:
![Screenshot from 2020-02-21 14-57-05](https://user-images.githubusercontent.com/44753941/75075188-7e6dd980-54ba-11ea-8b3f-92da66e06a68.png)

so yeah, one is bigger than the other. I guess it's OK? The mockup on Zeplin asked for the same size though.

Reply:
You're right:

![Screen Shot 2020-02-21 at 2 58 25 PM](https://user-images.githubusercontent.com/2294397/75075260-a5c4a680-54ba-11ea-91a5-89f9f3cd21ae.png)

That _does_ look kinda dumb. `rem`s it is!